zuul.d/jobs.yaml

```diff
@@ -530,8 +530,8 @@
       # constraints have pinning to install the right version of python-libjuju
       # based on what snap channel is being used to bootstrap the controller
       # from.
-      juju_snap_channel: '3.5/stable'
-      pip_constraints_url: 'https://raw.githubusercontent.com/openstack-charmers/zaza-openstack-tests/master/constraints/constraints-noble.txt'
+      # juju_snap_channel: '3.5/stable'
+      # pip_constraints_url: 'https://raw.githubusercontent.com/openstack-charmers/zaza-openstack-tests/master/constraints/constraints-noble.txt'
```

Comment:
For noble, we're transitioning to juju 3.5/stable, and moving off from 2.9; thus we don't really want to revert this change.